docs/contributions/README.md

```diff
@@ -0,0 +1,73 @@
+# How to Contribute
+
+MDC React is an open source project, so we encourage contributions! If you are
+looking for a bug to fix, check out [Help Wanted Issues](https://github.com/material-components/material-components-web-react/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) on GitHub. Otherwise please open a [new issue](https://github.com/material-components/material-components-web-react/issues/new).
+
+## Fixing a Bug or Adding a Feature
+
+MDC React's goal is to build in parity of [MDC Web](https://github.com/material-components/material-components-web). MDCR's features should have a 1:1 mapping within the constraints of React. If there is part of the API you disagree with please open a [new issue](https://github.com/material-components/material-components-web-react/issues/new). Depending on the content it may fall to MDC Web's plate, which trickles to MDCR a release later.
```

Comment:
don't say plate, because not every English speaker knows that saying. Be more specific.